lib/TWCManager/Control/HTTPControl.py

```diff
@@ -46,14 +44,18 @@ def __init__(self, master):
 
         httpd = ThreadingSimpleServer(("", self.httpPort), HTTPControlHandler)
         httpd.master = master
-        self.master.debugLog(1, "HTTPCtrl", "Serving at port: " + str(self.httpPort))
+        self.debugLog(1, "Serving at port: " + str(self.httpPort))
         threading.Thread(target=httpd.serve_forever, daemon=True).start()
 
+    def debugLog(self, minlevel, message):
+        self.master.debugLog(minlevel, "HTTPCtrl", message)
+
```

Comment:
This is effectively undoing work previously done to remove the previous local definitions so that everything is centralised - let's not do this and instead reference the master function directly, this move came as a result of having to update every module previously to add a new parameter to debugLog, and doing this would put us back in the same position again in future.

Reply:
Same as my first comment. I like to local approach much more, than always writing the module.